scripts/prepare-genesis/default.nix

```diff
@@ -1,39 +1,39 @@
-{ localLib ? import ./../../lib.nix
+with (import ./../../lib.nix);
+
+{ stdenv, writeScript, writeScriptBin
+, python3, yaml2json, jq, coreutils, gnused
+
+, cardano-sl, cardano-sl-tools
+
+## Options
 , stateDir ? localLib.maybeEnv "CARDANO_STATE_DIR" "./state-demo"
-, config ? {}
-, system ? builtins.currentSystem
-, pkgs ? import localLib.fetchNixPkgs { inherit system config; }
-, gitrev ? localLib.commitIdFromGitRepo ./../../.git
 , systemStart ? 0
 , configurationKey ? "testnet_full"
 , configurationKeyLaunch ? "testnet_launch"
 , numCoreNodes ? 7
 }:
 
-with localLib;
 
 let
-  iohkPkgs = import ./../.. { inherit config system pkgs gitrev; };
-
-  python = pkgs.python3.withPackages (ps: [ ps.pyyaml ]);
+  python = python3.withPackages (ps: [ ps.pyyaml ]);
 
-  yaml2json = pkgs.writeScriptBin "yaml2json" ''
+  yaml2json = writeScriptBin "yaml2json" ''
     #!${python}/bin/python
     import json
     import sys
     import yaml
     json.dump(yaml.load(open(sys.argv[1])), sys.stdout)
```

Comment:
`pkgs.haskell.lib.disableCabalFlag pkgs.haskellPackages.yaml "no-exe"` will give you a derivation with `yaml2json` and `json2yaml` binaries, which do the exact same job